### ISSUE TITLE
Clientside import not working with `svelte-leafletjs`?

### ISSUE THREAD

**Issue by cyqsimon** [bluwy/vite-plugin-iso-import#10]
I was trying to use `svelte-leafletjs` in my sveltekit project but ran into SSR import issues. So I followed the steps in [this issue](https://github.com/ngyewch/svelte-leafletjs/issues/3) to fix it. However, it seems like the clientside-only import is just not working for me. Am I doing something wrong?

## MRE
`/src/lib/MyMap.svelte`:

```svelte
<script lang="ts">
    import { browser } from "$app/environment";
    import { LeafletMap } from "svelte-leafletjs?client";
</script>

{#if browser}
    <LeafletMap />
{/if}
```

`/src/routes/+page.svelte`:

```svelte
<script lang="ts">
    import MyMap from "$lib/MyMap.svelte";
</script>

<MyMap />
```

```
[vite] Error when evaluating SSR module /node_modules/svelte-leafletjs/dist/svelte-leafletjs.js?client: failed to import "leaflet"
|- ReferenceError: window is not defined
    at /path/to/repo/node_modules/leaflet/dist/leaflet-src.js:230:19
    at /path/to/repo/node_modules/leaflet/dist/leaflet-src.js:7:66
    at Object.<anonymous> (/path/to/repo/node_modules/leaflet/dist/leaflet-src.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1368:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
    at Module.load (node:internal/modules/cjs/loader:1205:32)
    at Module._load (node:internal/modules/cjs/loader:1021:12)
    at cjsLoader (node:internal/modules/esm/translators:366:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:315:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)

[vite] Error when evaluating SSR module /src/lib/MyMap.svelte: failed to import "/node_modules/svelte-leafletjs/dist/svelte-leafletjs.js?client"
|- ReferenceError: window is not defined
    at /path/to/repo/node_modules/leaflet/dist/leaflet-src.js:230:19
    at /path/to/repo/node_modules/leaflet/dist/leaflet-src.js:7:66
    at Object.<anonymous> (/path/to/repo/node_modules/leaflet/dist/leaflet-src.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1368:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
    at Module.load (node:internal/modules/cjs/loader:1205:32)
    at Module._load (node:internal/modules/cjs/loader:1021:12)
    at cjsLoader (node:internal/modules/esm/translators:366:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:315:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)

[vite] Error when evaluating SSR module /src/routes/+page.svelte: failed to import "/src/lib/MyMap.svelte"
|- ReferenceError: window is not defined
    at /path/to/repo/node_modules/leaflet/dist/leaflet-src.js:230:19
    at /path/to/repo/node_modules/leaflet/dist/leaflet-src.js:7:66
    at Object.<anonymous> (/path/to/repo/node_modules/leaflet/dist/leaflet-src.js:10:3)
    at Module._compile (node:internal/modules/cjs/loader:1368:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
    at Module.load (node:internal/modules/cjs/loader:1205:32)
    at Module._load (node:internal/modules/cjs/loader:1021:12)
    at cjsLoader (node:internal/modules/esm/translators:366:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:315:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
```

This is the same error I was getting before using `vite-plugin-iso-import`.

## Versions

node: 21.7.0
sveltekit: 2.5
vite: 5.0.12
vite-plugin-iso-import: 1.1.0

**Comment by bluwy** [comment]
It works fine for me. Did you add the plugin in the Vite config file? https://stackblitz.com/edit/sveltejs-kit-template-default-krmi5f?file=vite.config.js

**Comment by cyqsimon** [comment]
> It works fine for me. Did you add the plugin in the Vite config file? https://stackblitz.com/edit/sveltejs-kit-template-default-krmi5f?file=vite.config.js

Ah okay thanks for the pointer. The import is working as expected now (although it's now throwing `Unrecoverable HMR error in <Root>`; but that seems to be a Svelte thing...).

My mistake was I followed [the steps here](https://github.com/ngyewch/svelte-leafletjs/issues/3#issuecomment-1170333938), but TS yelled at me because it seems like `kit.vite.plugins` in `svelte.config.js` has been deprecated and removed. But I found another config key in `svelte.config.js` called `vitePlugin`, so I simply declared `vitePlugin: [isoImport()]` assuming this would do it (and TS didn't complain). Apparently it doesn't (which I don't really understand why; nor why there are two places to configure Vite plugins in the first place).

Anyways, thanks for the extra-expedient response. I'll close the issue for now.
